### DISCUSSION TITLE
overview/jdk-install-config

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#214] [Announcements]
# overview/jdk-install-config

本文详细介绍了在Windows和macOS平台上安装JDK的两种方法：使用原始安装包手动配置环境变量和通过包管理器（如Chocolatey和Homebrew）一键式安装。我们将对比这两种方法的优缺点，帮助您选择最适合自己的JDK安装方式。

https://tobebetterjavaer.com/overview/jdk-install-config.html

**Comment by upeng** [comment]
oracle jdk 是不是收费的？不要在公司内网安装，否则会涉嫌商用，从安装之日起收费，我已经踩坑了，题主最好也讲下这方面的注意事项。ps: 我就是通过题主给的官方下载链接下载安装的...

**Reply by CharlesShan-hub** [reply]
我去这么坑！幸亏你说了
